flit/common.py

```diff
@@ -234,7 +234,7 @@ class Metadata:
     def __init__(self, data):
         self.name = data.pop('name')
         self.version = data.pop('version')
-        self.author_email = data.pop('author_email')
+        self.author_email = data.pop('author_email', None)
```

Comment:
Let's handle this like the other optional fields - remove this line, and set `author_email = None` on the class.